api_app/connectors_manager/models.py

```diff
@@ -1,9 +1,38 @@
 from django.db import models
+from django.contrib.postgres import fields as postgres_fields
+
+from api_app.models import Job
+
+
+HEALTH_STATUS_CHOICES = [
+    ("healthy", "healthy"),
+    ("failing", "failing"),
+    ("unknown", "unknown"),
+]
 
 
 class Connector(models.Model):
-    pass
+    name = models.CharField(max_length=128)
+    active = models.BooleanField(default=False)
+    description = models.TextField()
+    python_module = models.CharField(max_length=50)
+    config = postgres_fields.JSONField()
+    health_status = models.CharField(max_length=15, choices=HEALTH_STATUS_CHOICES)
+    health_checked_at = models.DateTimeField(blank=True)
+    soft_time_limit = models.PositiveIntegerField()
+
+    def __str__(self):
+        return self.name
 
 
 class ConnectorReport(models.Model):
-    pass
+    connector = models.ForeignKey(
+        Connector, related_name="reports", on_delete=models.CASCADE
+    )
+    job = models.ForeignKey(
+        Job, related_name="connector_reports", on_delete=models.CASCADE
+    )
+    result = postgres_fields.JSONField()
```

Comment:
rename `result` to `report` and add fields like start_time, end_time, status ("failed", "running", "success").

Reply:
@sp35